### PR TITLE
[DE] Add game modifiers `10K`, `Auto` and `Autopilot`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
After the related terminology was adjusted and approved in #8043 and #8079, now it is time to translate the missing game modifier articles. This PR is part 1 of the coming PR series.

## Translation notes

- ~~`hold bar` is meant to be `hold note`, thus this should be translated with `Hold-Note`~~
    - [x] requires #8159, awaits negotiation on the correct terminology

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Game_modifier/Auto/de.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+![AT Modsymbol](/wiki/shared/mods/AT.png "Auto (AT) Modsymbol")
+
+*Für die vollständige Liste aller Mods, siehe: [Spielmodifikationen](/wiki/Game_modifier)*\
+*Nicht zu verwechseln mit [Autopilot (Mod)](/wiki/Game_modifier/Autopilot).*
+
+## Übersicht
+
+- Abkürzung: AT oder AO
+- Typ: Sonstige
+- Score-Multiplikator: 1,00 x
```
Comment (The-Last-Cookie) [inline wiki/Gameplay/Game_modifier/Auto/de.md:25]: Now that I think about it, in my opinion, it's better to not put a space between the number and the x. The article `Co-op mod` has a table with numerous multipliers and I think putting a space everywhere makes the table less readable.

On the contrary, where I do think it to make sense is in the description of dimensions with the so called `Malkreuz` (e.g. `1920 × 1080`).

Very nitpicky, I know, but I wanted to mention it.

**Comment by Walavouchey** [comment]
it's "hold note" and not "hold bar", if that's relevant in any way.

File: wiki/Gameplay/Game_modifier/Auto/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Gameplay/Game_modifier/Auto/de.md:25]:
```suggestion
- Score-Multiplikator: 1,00x
```

File: wiki/Gameplay/Game_modifier/Auto/de.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+In [osu!taiko](/wiki/Game_mode/osu!taiko) wird die Mod Auto versuchen, einen perfekten Durchlauf zu erzielen, aber erlangt nicht immer die Note SS. Sie wird gelegentlich einen Tick bei Drumrolls, die nicht auf einem ganzen oder halben Takt starten, verfehlen und wird nicht immer in der Lage dazu sein, Dendens auf Beatmaps mit einer sehr hohen allgemeinen Schwierigkeit zu vervollständigen.
+
+### osu!catch
+
+In [osu!catch](/wiki/Game_mode/osu!catch) geht die Mod Auto über perfekt hinaus und nutzt unbegrenzte Geschwindigkeit, um *jede einzelne Banane* in einem Spinner zu fangen, obwohl eine solche Leistung für einen menschlichen Spieler unmöglich ist.
+
+### osu!mania
+
+In [osu!mania](/wiki/Game_mode/osu!mania) wird die Mod Auto versuchen, einen perfekten Durchlauf zu erzielen, aber verfehlt möglicherweise dicht aneinander gedrängte Noten (z. B. mehrere Tastendrücke nah beieinander auf einer Leiste).
```
Comment (The-Last-Cookie) [inline wiki/Gameplay/Game_modifier/Auto/de.md:52]:
```suggestion
In [osu!mania](/wiki/Game_mode/osu!mania) wird die Mod Auto versuchen, einen perfekten Durchlauf zu erzielen, aber verfehlt möglicherweise dicht aneinanderliegende Noten (z. B. mehrere Tastendrücke nah beieinander auf einer Hold-Note).
```

File: wiki/Gameplay/Game_modifier/Auto/de.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## Trivia
+
+- Beim Zuschauen eines Replays mit der Mod Auto ist der in der Rangleiste angezeigte Nutzername "osu!".
+- After watching an Auto replay, the mod will deselect itself and the user will be forced to reselect the mod if they wish to watch the replay again. This will happen whether or not the replay is watched all the way through.
```
Comment (The-Last-Cookie) [inline wiki/Gameplay/Game_modifier/Auto/de.md:57]:
```suggestion
- Nachdem ein Replay mit der Mod Auto angesehen wurde, wird die Mod automatisch deaktiviert und der Nutzer wird gezwungen, die Mod erneut anzuwählen, wenn er das Replay noch einmal anschauen möchte. Dies geschieht unabhängig davon, ob das Replay bis zum Ende angesehen wird oder nicht.
```

File: wiki/Gameplay/Game_modifier/Autopilot/de.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+![AP Modsymbol](/wiki/shared/mods/AP.png "Auto Pilot (AP) Modsymbol")
+
+*Für die vollständige Liste aller Mods, siehe: [Spielmodifikationen](/wiki/Game_modifier)*\
+*Nicht zu verwechseln mit [Auto (Mod)](/wiki/Game_modifier/Auto).*
+
+## Übersicht
+
+- Abkürzung: AP
+- Typ: Sonstige
+- Score-Multiplikator: 0,00 x
```
Comment (The-Last-Cookie) [inline wiki/Gameplay/Game_modifier/Autopilot/de.md:23]:
```suggestion
- Score-Multiplikator: 0,00x
```

File: wiki/Gameplay/Game_modifier/Auto/de.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+In [osu!catch](/wiki/Game_mode/osu!catch) geht die Mod Auto über perfekt hinaus und nutzt unbegrenzte Geschwindigkeit, um *jede einzelne Banane* in einem Spinner zu fangen, obwohl eine solche Leistung für einen menschlichen Spieler unmöglich ist.
+
+### osu!mania
+
+In [osu!mania](/wiki/Game_mode/osu!mania) wird die Mod Auto versuchen, einen perfekten Durchlauf zu erzielen, aber verfehlt möglicherweise dicht aneinander gedrängte Noten (z. B. mehrere Tastendrücke nah beieinander auf einer Leiste).
+
+## Trivia
+
+- Beim Zuschauen eines Replays mit der Mod Auto ist der in der Rangleiste angezeigte Nutzername "osu!".
```
Comment (The-Last-Cookie) [inline wiki/Gameplay/Game_modifier/Auto/de.md:56]:
```suggestion
- Beim Zuschauen eines Replays mit der Mod Auto ist der in der Rangliste angezeigte Nutzername "osu!".
```

**Comment by The-Last-Cookie** [comment]
> it's "hold note" and not "hold bar", if that's relevant in any way.

Thanks, I adjusted this in the self-review above.

File: wiki/Gameplay/Game_modifier/Auto/de.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+In [osu!catch](/wiki/Game_mode/osu!catch) geht die Mod Auto über perfekt hinaus und nutzt unbegrenzte Geschwindigkeit, um *jede einzelne Banane* in einem Spinner zu fangen, obwohl eine solche Leistung für einen menschlichen Spieler unmöglich ist.
+
+### osu!mania
+
+In [osu!mania](/wiki/Game_mode/osu!mania) wird die Mod Auto versuchen, einen perfekten Durchlauf zu erzielen, aber verfehlt möglicherweise dicht aneinanderliegende Noten (z. B. mehrere Tastendrücke nah beieinander auf einer Hold-Note).
+
+## Trivia
+
+- Beim Zuschauen eines Replays mit der Mod Auto ist der in der Rangliste angezeigte Nutzername "osu!".
```
Comment (FamousPig) [inline wiki/Gameplay/Game_modifier/Auto/de.md:56]:
```suggestion
- Beim Anschauen eines Replays mit der Mod Auto ist der in der Rangliste angezeigte Nutzername "osu!".
```

`Zuschauen` to me means watching something live. Watching a replay live sounds weird to me. Not a huge deal though.

**Review by FamousPig — CHANGES_REQUESTED** [review]
Looks good apart from a minor point (+ the unresolved todo item)

**Review by TicClick — APPROVED** [review]

**Pull request merged**